pkgs/stdenv/generic/check-meta.nix

```diff
@@ -80,14 +80,44 @@ let
   pos_str = meta: meta.position or "«unknown-file»";
 
   remediation = {
-    unfree = remediate_whitelist "Unfree";
+    unfree = remediate_unfree;
     broken = remediate_whitelist "Broken";
     unsupported = remediate_whitelist "UnsupportedSystem";
     blacklisted = x: "";
     insecure = remediate_insecure;
     broken-outputs = remediateOutputsToInstall;
     unknown-meta = x: "";
   };
+  remediate_unfree = attrs:
+    ''
+      a) For `nixos-rebuild` you can allow unfree packages globally:
+        { nixpkgs.config.allowUnfree = true; }
+
+      Or configure a predicate to whitelist specific packages:
+        { nixpkgs.config.allowUnfreePredicate = pkg: builtins.elem (lib.getName pkg) [
+            "package-name-to-whitelist"
```

Comment:
we could actually include the package name here, i think. `attrs.name`

Reply:
Great idea, done!